articles/142_idl.md

```diff
@@ -35,3 +35,45 @@ Further it describes how the interfaces are used to generate code in C, C++ and
 <div class="alert alert-warning" markdown="1">
   <b>TODO:</b> All of this too
 </div>
+
+### Type Mapping
+
+IDL types mapped to different types in different languages.
+
+
+| IDL type | C type    | C++ type    | Python type          |
```

Comment:
There seem to be differences between IDL types and DDS types. Would it make sense to have a DDS Type column here?

Reply:
By DDS types do you mean DDS-XTYPES 1.2?

Reply:
Yes I was thinking of DDS-XTYPES, to have a reference of what will end up on the wire when using a given IDL type.